resources/assets/js/components/Elements/Conversation.vue

```diff
@@ -0,0 +1,118 @@
+<template>
+    <div class="card">
+        <div class="card-header">{{this.sender}}</div>
+        <div class="card-body">
+            <div v-for="message in messages" :class="{'text-right': isCurrent(message.user_id)}"
+                 class="message-wrapper">
+            <span class="message"
+                  :class="[isCurrent(message.user_id) ? 'current-user' : 'sender']">{{message.text}}</span>
+                <span class="time" :class="[isCurrent(message.user_id) ? 'current-user-time' : 'sender-time']">{{message.date}} {{message.time}}</span>
+            </div>
+        </div>
+        <div class="card-footer">
+            <ajax-form event="sent" ref="form"
```

Comment:
we need to clear the input of the form once the button is clicked

Reply:
done